setup.cfg

```diff
@@ -41,31 +41,6 @@ install_requires =
     websockets>=9.0
     traitlets>=5.0.3,<6
 
-[options.extras_require]
-dev =
-    black
-    bump2version
```

Comment:
Do we need to remove all `extras_require`?

Maybe we can keep `dev`, removing it might cause issues with the `bump-version.py` script if some dependencies are missing.

Reply:
Actually `check_release` seems to be passing: https://github.com/voila-dashboards/voila/actions/runs/3144155823/jobs/5109792215

So maybe it's fine this way.

Reply:
I merged all extras into one requirements.txt file. If you see a benefit for this, I could split it into multiple files?

Reply:
I meant if it's only the direct dependency on `ipyvolume` causing issues then maybe only the `visual_test` dependencies could be moved to a different file.